test/cmd/lfstest-gitserver.go

```diff
@@ -76,6 +76,14 @@ func main() {
 			return
 		}
 
+		if strings.Contains(r.URL.Path, "/info/lfs/locks") {
```

Comment:
Does this handle unlocking? We should probably remove the two old handlers on L71-72 to make sure that we're hitting the right URL.